packages/replayio/src/commands/view.ts

```diff
@@ -0,0 +1,14 @@
+import { spawn } from "child_process";
+import { registerCommand } from "../utils/commander";
+import { exitProcess } from "../utils/exitProcess";
+import { getSystemOpenCommand } from "../utils/getSystemOpenCommand";
+
+registerCommand("view <id>").description("Upload one or more recordings").action(view);
+
+async function view(id: string) {
+  const url = `https://app.replay.io/recording/${id}`;
```

Comment:
We'll have to make this (and similar places) configurable somehow to aid testing against cloud dev.

Reply:
I found that you already have `replayAppHost` here so this could be used here

Reply:
Yup! This has already been made testable via the `REPLAY_APP_SERVER` var in the next branch.